.travis.yml

```diff
@@ -7,6 +7,13 @@ jobs:
     - stage: test
       node_js: "12"
       os: linux
+      arch: amd64
+      before_script: npm run lint || exit 1;
```

Comment:
These scripts are only needed for the first build in this list. That build should be the linux build as it's the canonical.

Reply:
Not sure what you mean here, should I remove the arm64 job from the test stage?

Reply:
Yes please.

Reply:
Please maintain the job grouping by node version in the order linux, osx, arm64